catalogs/catalog-lakehouse-hudi/build.gradle.kts

```diff
@@ -45,6 +45,7 @@ dependencies {
   implementation(libs.commons.configuration1)
   implementation(libs.htrace.core4)
   implementation(libs.guava)
+  implementation(libs.commons.io)
```

Comment:
Please put it after `implementation(libs.commons.configuration1)` to follow alphabetical order.